Frameworks/UIKit/UIViewController.mm

```diff
@@ -1818,9 +1818,7 @@ - (void)_loadXamlPage {
     // Walk the list of outlets and assign them to the corresponding XAML UIElement
     unsigned int propListCount = 0;
     objc_property_t* props = (objc_property_t*)class_copyPropertyList([self class], &propListCount);
-    auto freeProps = wil::ScopeExit([&]() {
-        IwFree(props);
-    });
+    auto freeProps = wil::ScopeExit([&]() { IwFree(props); });
```

Comment:
Clang format I assume? I actually think its more readable the other way but no big deal.